.circleci/config.yml

```
@@ -0,0 +1,107 @@
+version: 2
+
+jobs:
+  requirements:
+    docker:
+      - image: web3f/ansible:v0.0.2
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          command: |
+            pip install -r w3f/substrate-benchmarks/requirements.txt
+
+  benchmark-1:
+    docker:
+      - image: web3f/ansible:v0.0.2
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          command: |
+            ansible-playbook --extra-vars "set_machine_type=e2-micro" w3f/substrate-benchmarks/benchmarks.yaml
+      - store_artifacts:
+          path: w3f/substrate-benchmarks/results
+          destination: results-artifact-e2-micro
+
+  benchmark-2:
+    docker:
+      - image: web3f/ansible:v0.0.2
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          command: |
+            ansible-playbook --extra-vars "set_machine_type=e2-small" w3f/substrate-benchmarks/benchmarks.yaml
+      - store_artifacts:
+          path: w3f/substrate-benchmarks/results
+          destination: results-artifact-e2-small
+
+  benchmark-3:
+    docker:
+      - image: web3f/ansible:v0.0.2
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          command: |
+            ansible-playbook --extra-vars "set_machine_type=e2-medium" w3f/substrate-benchmarks/benchmarks.yaml
+      - store_artifacts:
+          path: w3f/substrate-benchmarks/results
+          destination: results-artifact-e2-medium
+
+  benchmark-4:
+    docker:
+      - image: web3f/ansible:v0.0.2
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          command: |
+            ansible-playbook --extra-vars "set_machine_type=e2-standard-2" w3f/substrate-benchmarks/benchmarks.yaml
+      - store_artifacts:
+          path: w3f/substrate-benchmarks/results
+          destination: results-artifact-e2-standard-2
+
+workflows:
+  version: 2
+  test_and_deploy:
+    jobs:
+      - requirements:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*/
+      - benchmark-1:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*/
+          requires:
+            - requirements
+      - benchmark-2:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*/
+          requires:
+            - benchmark-1
```

Why not running all the benchmarks in parallel?
```suggestion
            - requirements
```

---

bc we use the same compute instance name. I will refactor.